src/app/counter/wrapperTest.ts

```diff
@@ -1,17 +1,17 @@
-import { interval } from 'rxjs';
+// import { interval } from 'rxjs';
```

Comment:
Just remove files you don't need. If you want the code as documentation, move it to a markdown file. If you just want to find it back later, tag the commit it is in and let version control look after it :)